app/views/admin/words/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% provide(:title, "All Words")%>
+<div class=container>
+    <div class=row>
+        <h1> All Words </h1>
+        <%= link_to "Add Word", new_admin_category_word_path , class: 'btn btn-primary' %>
+        <%= will_paginate %>
+        <table>
```

Comment:
@johnedelmapa Display category info in this page, and provide a back button to category list for better UX.

Reply:
@johnedelmapa
- Back button is okay
- But please add category info, in current display, we could not tell which category it belonged

![image](https://user-images.githubusercontent.com/20507565/61683683-06370e00-ad49-11e9-8a14-487020799e5f.png)